ibis/spark/compiler.py

```diff
@@ -0,0 +1,43 @@
+import ibis.sql.compiler as comp
+from ibis.impala.compiler import (
+    ImpalaContext,
+    ImpalaDialect,
+    ImpalaExprTranslator,
+    ImpalaSelect,
+)
+
+
+class SparkSelectBuilder(comp.SelectBuilder):
+    @property
+    def _select_class(self):
+        return SparkSelect
+
+
+class SparkUnion(comp.Union):
```

Comment:
Do we need this class? I believe we established that Spark's `UNION` construct behaves in the typical SQL way.